lib/controllers/pharmacies.js

```diff
@@ -4,6 +4,8 @@ var express         = require("express"),
     auth            = require("./helpers/auth.js"),
     list            = require("./helpers/list.js");
 
+const guard = auth.roleGuard;
```

Comment:
NOTE: this functionality is not currently being used in the Orange app

Reply:
@B3rry @mountHouli Should we disable these endpoints entirely?